Habitica/src/main/java/com/habitrpg/android/habitica/ui/AvatarWithBarsViewModel.kt

```diff
@@ -65,10 +66,10 @@ class AvatarWithBarsViewModel(private val context: Context, view: View, userRepo
         avatarView.setAvatar(user)
 
         if (stats.habitClass != null) {
-            userClass = stats.getTranslatedClassName(context)
+            userClass = getTranslatedClassName(stats, context)

```

Comment:
Due to the way that the shared library handles resources (even for the native side of the shared code) the translated class name had to be extracted to a helperclass

Reply:
The stats class can take context as a parameter, however access to the resources of the habitRPG project will fail. I think it might be possible to migrate the HabitRPG resources into the shared project, but I would do that as a second PR. 
In fact, I could write that branch and attempt to merge it with this one so the changes are distinct and we could review that separately